Persistence/tables.sql

```diff
@@ -0,0 +1,67 @@
+CREATE DATABASE brewmes;
+
+CREATE TABLE Vibration (
+    id SERIAL PRIMARY KEY,
+    time_stamp TIME,
+    vibration FLOAT
+);
+
+CREATE TABLE Temperature (
+    id SERIAL PRIMARY KEY,
+    time_stamp TIME,
+    temperature FLOAT
+);
+
+CREATE TABLE Humidity (
+    id SERIAL PRIMARY KEY,
+    time_stamp TIME,
+    humidity FLOAT
+);
+
+CREATE TABLE Time_in_states (
+    id SERIAL PRIMARY KEY,
+    time_stamp TIME,
+    _state INT --bad naming but state is a reserved word

```

Comment:
Consider renaming to 'machine_state', for clarity

Reply:
it is done 🦕